terraform/infrastructure/iam/aws/main.tf

```diff
@@ -54,17 +54,14 @@ resource "aws_iam_policy" "control_plane_policy" {
     {
       "Effect": "Allow",
       "Action": [
```

Comment:
Why does this go into the control plane policy and not into `alb_policy.json`?

Reply:
Good point. It should go there!
I just attached it to the control plane in the first place because I saw that the relevant controller pod ran on the control plane. Will adjust, and possibly incorporate this into a generate target.